power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/vnf_pgm_converter.h

```diff
@@ -13,29 +13,28 @@ extern "C" {
 #endif
 
 /**
- * @brief Create the PGM_VNF_converter
+ * @brief Create the PGM_IO_VnfConverter
  * @param handle
  * @param file_buffer A pointer to the null-terminated C string.
- * @return The pointer to a PGM_VNF_converter instance. The instance must be freed by
- * PGM_VNF_delete_Converter
+ * @return The pointer to a PGM_IO_VnfConverter instance. The instance must be freed by
+ * PGM_IO_delete_vnf_converter.
  */
-PGM_IO_API PGM_IO_VnfConverter* PGM_VNF_create_converter(PGM_IO_Handle* handle, char* file_buffer);
+PGM_IO_API PGM_IO_VnfConverter* PGM_IO_create_vnf_converter(PGM_IO_Handle* handle, char* file_buffer);
 
 /**
  * @brief Retrieve the transformed input data from .vnf format to PGM format
  * @param handle
- * @param converter_ptr A pointer to a PGM_VNF_converter instace.
+ * @param converter_ptr A pointer to a PGM_IO_VnfConverter instace.
  * @param dataset A pointer to the const dataset supplied by the user.
  * @return The pointer to the const dataset instance supplied by the user which has been filled in.
  */
-PGM_IO_API PGM_IO_ConstDataset const* PGM_VNF_get_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr,
-                                                             PGM_IO_ConstDataset const* dataset);
+PGM_IO_API char const* PGM_IO_get_vnf_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr);
 
 /**
- * @brief Destroy the converter and free up the memory that was dedicated to it.
- * @param converter_ptr A pointer to a PGM_VNF_converter instance.
+ * @brief Destroy the PGM_IO_VnfConverter and free up the memory that was dedicated to it.
+ * @param converter_ptr A pointer to a PGM_IO_VnfConverter instance.
  */
-PGM_IO_API void PGM_VNF_delete_Converter(PGM_IO_VnfConverter* converter_ptr);
+PGM_IO_API void PGM_IO_delete_vnf_converter(PGM_IO_VnfConverter* converter_ptr);
```

Comment:
consistency with PGM terminology
```suggestion
PGM_IO_API void PGM_IO_destroy_vnf_converter(PGM_IO_VnfConverter* converter_ptr);
```